### PR TITLE
Clear button in article publish date

### PR DESCRIPTION
**Pull request by akritianand**
Pull Request for Issue #17744.

### Summary of Changes
Previously, when we edit an article under publishing tab, the clear button in calendar field did not work if it was a required field.
Now, it clears the calendar field when pressed instead of redirecting (as it did originally).
Changes were made in calendar.min.js

### Testing Instructions
Create an admin form with a required date field. i.e. open administrator\components\com_content\models\forms\article.xml and make field "publish_down" required by adding required="true" to field's xml node.
Go to Joomla! administration panel and open an article edit view.
Finish Publishing field should now be a required field.
Add Finish Publishing date, using the date picker of the calendar
Click "Clear" button on the Calendar.

### Expected result
Clears the field

### Actual result
Redirects to Article Page

### Documentation Changes Required
None

### CODE REVIEW

**Comment by ReLater** [comment]
Sorry! I thought this is an issue (not PR): Deleted my comment

**Comment by dgrammatiko** [comment]
Can you please include the changes in the non minified version? The minified is not reviewable and also it's just the minified version of calendar.js

**Comment by akritianand** [comment]
@dgt41 can you review again. I've made the changes you asked.

**Comment by dgrammatiko** [comment]
@akritianand seems fine, thanks

**Comment by ghost** [comment]
@akritianand can you please look if #17744 is same PR?

**Comment by akritianand** [comment]
@franz-wohlkoenig oh sorry, I think it's the same issue that I was solving. Can that previous patch be removed or closed in any way?

**Comment by ghost** [comment]
@akritianand sorry, meant #17793

**Comment by akritianand** [comment]
@franz-wohlkoenig Yes, I meant the same.

**Comment by ghost** [comment]
~~@maintainers: Looks like PR #17793 and this are about same Issue - can you please decide how to handle this?~~
#17793 is closed

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on af24b7347d36cd1c50fcadaa3f3e2758fb1b5cab<br /><br />Hit "Clear" on required Field as Example given above clears Field.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17809">issues.joomla.org/tracker/joomla-cms/17809</a>.</sub>

File: administrator/components/com_modules/models/module.php [hunk 1]
```diff
@@ -927,7 +927,8 @@ public function save($data)
 
 			if ($data['title'] == $orig_table->title)
 			{
-				$data['title'] .= ' ' . JText::_('JGLOBAL_COPY');
+				//$data['title'] .= ' ' . JText::_('JGLOBAL_COPY');
```
Comment (Quy) [inline administrator/components/com_modules/models/module.php]: Please delete line. 

From Joomla Coding Standards:
`Do not leave commented code unless there is a clearly explained reason for doing so. In such a case, a comment saying "Code intentionally commented" will prevent accidental removal.`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on af24b7347d36cd1c50fcadaa3f3e2758fb1b5cab<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17809">issues.joomla.org/tracker/joomla-cms/17809</a>.</sub>

**Comment by Quy** [comment]
I will do a PR to delete the commented line after the merge if the OP does not do it beforehand.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17809">issues.joomla.org/tracker/joomla-cms/17809</a>.</sub>

**Comment by infograf768** [comment]
GRR:
https://github.com/joomla/joomla-cms/pull/22026

😭

**Pull request merged**